adapter/internal/discovery/xds/enforcercallbacks/enforcer_callbacks.go

```diff
@@ -44,10 +46,12 @@ func (cb *Callbacks) OnStreamClosed(id int64) {
 
 // OnStreamRequest prints debug logs
 func (cb *Callbacks) OnStreamRequest(id int64, request *discovery.DiscoveryRequest) error {
+	nodeIdentifier := getNodeIdentifier(request)
```

Comment:
shall we add a log level check before this, as this is only used for debugging

Reply:
This is not valid after the new change we brought in where we print a new info log entry to identify the connected router enforcer pods.